### DISCUSSION TITLE
Possible to display events in weeks / columns + card size ?

### DISCUSSION THREAD

**Discussion by cedricdelecole** [totaldebug/atomic-calendar-revive#1597] [General]
Hello, 

Looking for displaying a calendar on my wall mounted tablet, (google calendar), where I can : 
1. Display the events in 5 or 7 columns, showing weekdays of the week
2. rapt the width and height of the card so that it fits perfectly on my wall mounted tablet ...

Is it possible with this calendar ?
Many thanks !!

**Comment by marksie1988** [comment]
No that is not possible, only the calendar and event view exist
